Should help a bit vs. https://github.com/kuthulux/gnome-connection-manager/issues/4
that the pane separator is too hard to catch with the mouse.

--- github/gnome-connection-manager.glade	2020-04-14 18:33:04.000000000 +0000
+++ fixed/gnome-connection-manager.glade	2020-04-15 11:23:15.597470553 +0000
@@ -3096,6 +3096,7 @@
               <object class="GtkHPaned" id="hpMain">
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
+                <property name="wide_handle">True</property>
                 <signal name="button-press-event" handler="on_hpMain_button_press_event" swapped="no"/>
                 <child>
                   <object class="GtkScrolledWindow" id="scrolledwindow1">
